src/components/existing-request/existing-request-display.vue

```diff
@@ -792,7 +792,7 @@ export default class ExistingRequestDisplay extends Mixins(
 </script>
 
 <style lang="scss" scoped>
-@import "@/assets/scss/theme";
+@import "@/assets/styles/theme";
 
 .nr-data .col {
   color: $text;
```

Comment:
I don't know why this isn't showing as an error (`$text` is not declared). Does this work even with the weird import?

Reply:
Per comment above, this works, so this conversation is invalid and I'm going to resolve it.